pyproject.toml

```diff
@@ -4,15 +4,14 @@
 
 [project]
 name = "lcm"
-dynamic = ["version"]
-authors = [
-    { name = "Tim Mensinger", email = "mensingertim@gmail.com" },
-    { name = "Hans-Martin von Gaudecker", email = "hmgaudecker@uni-bonn.de" },
-    { name = "Janos Gabler" },
+description = "lcm aims to generalize and facilitate the specification, solving, and estimation of dynamic choice models."
+requires-python = ">=3.12"
+dependencies = [
```

Comment:
Can't we just add them as pixi-dependencies? I'd prefer to have everything in one place and jax is missing from here, anyhow.

Reply:
We can add them additionally to the pixi-dependencies, but I don't think we can remove them from there. When installing LCM via pip these dependencies will be installed alongside LCM. The packages managed via pixi are all dependencies that are only required for the development of LCM.

Jax and jaxlib are missing here because, for now, I want to force the user to install their own Jax/jaxlib versions. Depending on whether you want to use LCM on a CPU or GPU and depending on the operating system, you must follow specific installation processes to install the correct jaxlib version. Hence, I think the easiest route is not to list them as dependencies for now. I'd say that once the LCM user base grows, we will have to think about a more user-friendly approach.

Reply:
Just as most scientific packages require use of conda, I'd be fine requiring use of pixi. Happy to take bets that by the time or user base grows, it will be standard!